tests/framework/PostProcessors/DataClassifier/test_dataClassifier_postprocessor.xml

```diff
@@ -91,14 +91,14 @@
       <Input   class="DataObjects"  type="PointSet"        >ET_PS</Input>
       <Input   class="DataObjects"  type="PointSet"        >sim_PS</Input>
       <Model   class="Models"       type="PostProcessor"   >ET_Classifier</Model>
-      <Output  class="DataObjects"  type="PointSet"        >sim_PS</Output>
+      <Output  class="DataObjects"  type="PointSet"        >sim_PS_out</Output>
```

Comment:
It looks like the "revisions" weren't updated for this test.

Reply:
added, actually the test info is missing